docker-compose.data.yaml

```diff
@@ -0,0 +1,6 @@
+version: '3.8'
+
+services:
+  ollama:
+    volumes:
+      - ${OLLAMA_DATA_DIR-./ollama-data}:/root/.ollama
```

Comment:
```
volumes:
      - ollama:/root/.ollama
```

already exists in [docker-compose.yaml](https://github.com/ollama-webui/ollama-webui/pull/275/files#diff-3fde9d1a396e140fefc7676e1bd237d67b6864552b6f45af1ebcc27bcd0bb6e9), could you clarify the purpose of this yaml file? I believe it's best if we stick to `ollama:/root/.ollama` volume.

Reply:
it's by choice, on script run-compose.sh it's defined as default ollama:/root/.ollama.
Of course i'll correct the compose asap with ollama:/root/.ollama